templates/configmaps.yaml

```diff
@@ -3,28 +3,15 @@ kind: ConfigMap
 metadata:
   name: {{ template "fullname" . }}
   labels:
-    app: {{ template "fullname" . }}
-    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
-    release: {{ .Release.Name | quote }}
-    heritage: {{ .Release.Service | quote }}
+    {{- include "standard-labels" . | nindent 4 }}
 data:
 {{- if .Values.solr.enabled }}
   INDEX_HOST: {{ .Values.solr.host | quote  }}
   INDEX_PORT: {{ .Values.solr.port | quote  }}
 {{- end }}
-{{- if .Values.mysql.enabled }}
-  DB_USER: {{ .Values.mysql.mysqlUser | quote }}
-  DB_HOST: "{{ .Release.Name }}-mysql"
-  DB_DATABASE: {{ .Values.mysql.mysqlDatabase | quote }}
-{{- else if .Values.postgresql.enabled }}
-  DB_USER: {{ .Values.postgresql.postgresqlUsername | quote }}
-  DB_HOST: "{{ .Release.Name }}-postgresql"
-  DB_DATABASE: {{ .Values.postgresql.postgresqlDatabase | quote }}
-{{- else }}
-  DB_USER: {{ .Values.externalDB.user | quote }}
-  DB_HOST: {{ .Values.externalDB.host | quote }}
-  DB_DATABASE: {{ .Values.externalDB.database | quote }}
-{{- end }}
+  DB_USER: {{ template "databaseUser" . }}
+  DB_HOST: {{ template "databaseHost" . }}
+  DB_DATABASE: {{ template "databaseDatabase" . }}
```

Comment:
Are you certain you don't want `include` instead of `template`? Please double check this, they behave differently.

Same for any other place that uses `template`.